portal/portal-render-engine-impl/impl/src/webapp/vm/morpheus/snippets/skipNav-snippet.vm

```diff
@@ -15,18 +15,18 @@
                 <span class="accesibility_key">[c]</span>
             </a>
         </li>
-        <li role="menuitem" class="Mrphs-skipNav__menuitem Mrphs-skipNav__menuitem--tools">
-            <a href="#totoolmenu" class="Mrphs-skipNav__link js-toggle-tools-nav" title="${rloader.sit_jumptools}" accesskey="l">
-                <i class="fa fa-cubes tools-icon"></i>
-                ${rloader.sit_menutools}
-                <span class="accesibility_key">[l]</span>
-            </a>
-        </li>
         <li role="menuitem" class="Mrphs-skipNav__menuitem Mrphs-skipNav__menuitem--worksite">
             <a href="#sitetabs" id="more-sites-menu" class="Mrphs-skipNav__link js-toggle-sites-nav" title="${rloader.sit_jumpworksite}" accesskey="w">
                 <i class="fa fa-th all-sites-icon"></i> ${rloader.sit_worksites}
                 <span class="accesibility_key">[w]</span>
             </a>
         </li>
+        <li role="menuitem" class="Mrphs-skipNav__menuitem Mrphs-skipNav__menuitem--tools">
+            <a href="#totoolmenu" class="Mrphs-skipNav__link js-toggle-tools-nav" title="${rloader.sit_jumptools}" accesskey="l">
+                <i class="fa fa-bars tools-icon"></i>
```

Comment:
I think we need to aria-hidden="true" this FontAwesome icon if it has no important meaning for screenreader users